### PR TITLE
Chore/58/update docs

### LINKED ISSUE #58: Update docs to go over how to add the bot to only select channels
**Is your feature request related to a problem? Please describe.**
During development we will probably only want the bot to be available in certain channels rather than the entire server.
This can be done using discord itself, but this process should be documented.

**Describe the solution you'd like**
Update the README with information on limiting the discord bot to select channels.

### PR DESCRIPTION
**Pull request by naomi-lgbt**
I have added the instructions for granting/denying specific channel access, and updated some wording throughout the docs.

Closes #58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,15 +96,33 @@ If you don't have a Discord bot token, you'll have to create one. To create a Di
...
+For each channel you want keep the bot *out* of, you need to do the following:
+
+1. Select the gear icon next to the channel.
+2. Select "Permissions". 
+3. Next to the Roles/Members, click the `+` icon. 
+4. Add the bot's role.
+5. The added role should now be selected.
+6. To the right, go through the permissions list and set each permission to the red "X".
+   The most important permissions to turn off are "Read message history", "Read messages", and "Send messages".
+
```
Comment (bradtaniguchi) [inline README.md]: Add another step for the that mentions clicking "save changes"

**Review by bradtaniguchi — COMMENTED** [review]
Thanks for updating the grammar across the README too BTW.

Add one more step that mentions to hit save and we will be good!

File: README.md (hunk 1 above)
Comment (naomi-lgbt) [reply README.md]: Done and done. 
Though the Discord app won't let you exit that screen without either choosing "Save Changes" or "reset". It shakes and gets all mad at you 😛

Comment (bradtaniguchi) [reply README.md]: Best if we instruct people to save without making discord angry :P

**Pull request merged**